packages/bodiless-components/src/List/asBodilessList.tsx

```diff
@@ -45,26 +51,45 @@ const asTitledItem = <P extends TitledItemProps>(Item: ComponentType<P>) => {
   return TitledItem;
 };
 
+const asOverviewItem = <P extends TitledItemProps>(Item: ComponentType<P>) => {
+  const ItemWithOverview: ComponentType<P> = (props) => {
```

Comment:
this can be FC<P>

Reply:
See my comment where this is referenced - i think we should move it there.

Reply:
Done